src/uct/api/v2/uct_v2.h

```diff
@@ -228,6 +272,24 @@ uct_iface_estimate_perf(uct_iface_h tl_iface, uct_perf_attr_t *perf_attr);
 ucs_status_t uct_md_mem_dereg_v2(uct_md_h md,
                                  const uct_md_mem_dereg_params_t *params);
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Check if remote iface address is reachable.
+ *
+ * This function checks if a remote address can be reached from a local interface.
+ * If the function returns true, it does not necessarily mean a connection and/or
+ * data transfer would succeed, since the reachability check is a local operation
```

Comment:
comment is confusing - why does it not guarantee?

Reply:
It is duplicates the `uct_iface_is_reachable()` description:
https://github.com/openucx/ucx/blob/b75809faa3e8c2015c5f5ab529ae90b0402c7ff0/src/uct/api/uct.h#L1955-L1958
I assume that the `v2` version should repeat the orig description.

Reply:
@yosefe please correct me if this description requires modifications.